compose/runtime/runtime/src/wasmJsTest/kotlin/kotlinx/test/Actuals.wasm.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.test
+
+import kotlinx.coroutines.*
+
+@OptIn(DelicateCoroutinesApi::class)
+actual suspend fun testWithTimeout(timeoutMs: Long, block: suspend CoroutineScope.() -> Unit) {
+    GlobalScope.launch {
```

Comment:
```suggestion
    coroutineScope {
         block()
    }
```
Will this work? An async call is fragile.

Reply:
This function is not implemented properly at all. It ignores `timeoutMs`. I'll update it separately. 
It's used in our tests only.

Reply:
COMPOSE-662